src/components/ChatEntry.js

```diff
@@ -1,22 +1,29 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({id, sender, body, timeStamp, liked, onUpdateChat}) => {
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">
+          <TimeStamp time={timeStamp} />
+        </p>
+        <button className="like" onClick={() => onUpdateChat(id)}>{liked ? '❤️': '🤍'}</button>
```

Comment:
The ternary is pretty short here on line 15 so it's ok to embed it into the JSX. If it were a longer line of code, I'd suggest moving the logic into a separate line then storing the value in a variable and use the variable here